contrib/windows_installer/DOSBox-Staging-setup.iss

```diff
@@ -0,0 +1,130 @@
+Name: "czech"; MessagesFile: "compiler:Languages\Czech.isl"
+Name: "danish"; MessagesFile: "compiler:Languages\Danish.isl"
+Name: "dutch"; MessagesFile: "compiler:Languages\Dutch.isl"
+Name: "finnish"; MessagesFile: "compiler:Languages\Finnish.isl"
+Name: "french"; MessagesFile: "compiler:Languages\French.isl"
+Name: "german"; MessagesFile: "compiler:Languages\German.isl"
+Name: "hebrew"; MessagesFile: "compiler:Languages\Hebrew.isl"
+Name: "icelandic"; MessagesFile: "compiler:Languages\Icelandic.isl"
+Name: "italian"; MessagesFile: "compiler:Languages\Italian.isl"
+Name: "japanese"; MessagesFile: "compiler:Languages\Japanese.isl"
+Name: "norwegian"; MessagesFile: "compiler:Languages\Norwegian.isl"
+Name: "polish"; MessagesFile: "compiler:Languages\Polish.isl"
+Name: "portuguese"; MessagesFile: "compiler:Languages\Portuguese.isl"
+Name: "brazilianportuguese"; MessagesFile: "compiler:Languages\BrazilianPortuguese.isl"
+Name: "russian"; MessagesFile: "compiler:Languages\Russian.isl"
+Name: "slovak"; MessagesFile: "compiler:Languages\Slovak.isl"
+Name: "slovenian"; MessagesFile: "compiler:Languages\Slovenian.isl"
+Name: "spanish"; MessagesFile: "compiler:Languages\Spanish.isl"
+Name: "turkish"; MessagesFile: "compiler:Languages\Turkish.isl"
+Name: "ukrainian"; MessagesFile: "compiler:Languages\Ukrainian.isl"
+
+[Tasks]
+Name: "contextmenu"; Description: "Add ""Run/Open with {#DOSBoxAppName}"" context menu for Windows Explorer"
+Name: "defaultmsvc"; Description: "Run MSVC release binary (instead of MSYS2 binary) as the default binary"; Flags: unchecked
+Name: "desktopicon"; Description: "{cm:CreateDesktopIcon}"; GroupDescription: "{cm:AdditionalIcons}"
+
+[Run]
+Filename: "{app}\{#DOSBoxAppExeName}"; Parameters: "-c ""config -wc"" -noconsole"; Description: "{cm:LaunchProgram,{#StringChange(DOSBoxAppName, '&', '&&')}}"; Flags: nowait postinstall skipifsilent
+
+[Files]
+Source: "program\*"; DestDir: "{app}"; Flags: ignoreversion recursesubdirs createallsubdirs
+Source: "dosbox.exe"; DestDir: "{app}"; DestName: "dosbox.exe"; Flags: ignoreversion; Tasks: not defaultmsvc
+Source: "dosbox_with_debugger.exe"; DestDir: "{app}"; DestName: "dosbox_with_debugger.exe"; Flags: ignoreversion; Tasks: not defaultmsvc
+Source: "dosbox_msvc.exe"; DestDir: "{app}"; DestName: "dosbox.exe"; Flags: ignoreversion; Tasks: defaultmsvc
+Source: "dosbox_msvc_with_debugger.exe"; DestDir: "{app}"; DestName: "dosbox_with_debugger.exe"; Flags: ignoreversion; Tasks: defaultmsvc
+Source: "dosbox_with_console.bat"; DestDir: "{app}"; DestName: "dosbox_with_console.bat"; Flags: ignoreversion
+
+[Icons]
+Name: "{group}\{#DOSBoxAppName}"; Filename: "{app}\dosbox.exe"; Parameters: "-noconsole"
+Name: "{group}\{#DOSBoxAppName} with console"; Filename: "{app}\dosbox_with_console.bat"
+Name: "{group}\{#DOSBoxAppName} with debugger"; Filename: "{app}\dosbox_with_debugger.exe"
+Name: "{group}\Visit {#DOSBoxAppName} website"; Filename: "https://dosbox-staging.github.io/"
```

Comment:
The https URL can be replaced with the `{#DOSBoxAppURL}` define.

Reply:
Done replacing too.